gateways/clients/hathor_core_client.py

```diff
@@ -114,13 +115,14 @@ async def post(
 class HathorCoreClient:
     """Client to make requests
 
-    :param domain: domain where the requests will be made, defaults to config `hathor_core_domain`
-    :type domain: str, optional
+    :param url: url where the requests will be made, defaults to config `hathor_core_url`
+    :type url: str, optional
     """
 
-    def __init__(self, domain: Optional[str] = None) -> None:
-        self.domain = domain or HATHOR_CORE_DOMAIN
+    def __init__(self, url: Optional[str] = None, use_ssl: bool = True) -> None:
+        self.url = url or HATHOR_CORE_URL
         self.log = logger.new(client="sync")
+        self.protocol = "https" if use_ssl else "http"
```

Comment:
question: Should `protocol` be a private property? If so, I didn't see any use of it, which may be a bug.
If it's a public property, maybe add it to some docstrings as well.

Reply:
Actually this is a leftover, I've removed it: https://github.com/HathorNetwork/hathor-explorer-service/pull/326/commits/71588c0930c6dae82d77c50a2e24e7552ff3edcd